README.md

```diff
@@ -1,3 +1,71 @@
 # Hey Rodney dataset
 
 This is a dataset of voice samples for our wake word detection of "Hey Rodney".
+
+## Steps to Record audio data and upload it on github
+
+## Specifications:
+
+| __Phrase__ | "Hey Rhodney"   |
+| ------------- | ------------- |
+| __Number of Samples__  | 3  |
```

Comment:
Maybe it would be good to clarify here if you want to have three files, or three samples in a single file.